_posts/2016-08-30-bf-partnerships.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: Building open collaborations for the sustainable support of proprietary file formats
+categories: file-formats community
+---
+
+The challenges and cost associated with the development and maintenance of
+software for reading images stored in proprietary file formats (PFFs) have
+been discussed at length in previous blog posts 
+[1](http://blog.openmicroscopy.org/file-formats/2014/10/16/file-formats/), 
+[2](http://blog.openmicroscopy.org/file-formats/community/2016/01/06/format-support/).
+One approach to help address these issues is the development of  community
+collaborations that provide sustainable solutions to PFF support in
+Bio-Formats.
+ 
+In this blog post, we want to describe two successful examples of partnerships
+established with well recognized commercial entities,
+[Carl Zeiss Microscopy GmbH](http://www.zeiss.com/) and
+[Intelligent Imaging Innovations (“3i”)](https://www.intelligent-imaging.com/)
+and how this will result in more open, reusable code and better tools for the
+imaging community.
+ 
+## ZEISS: Open support for JPEG-XR compression
+ 
+Some image acquisitions systems built by Carl Zeiss Microscopy GmbH store
+binary image data using [JPEG-XR](https://jpeg.org/jpegxr/) compression.
+Open-source efforts to decode data stored using this technology have to date
+been unsuccessful. As we have noted previously, the complexity of providing a
+pure Java implementation of this compression scheme was simply to high to be
+fully assumed by a non-profit, grant funded organization like OME.
+ 
+In response, the user community has raised their concerns to ZEISS and a
+partnership has been established with
+[Glencoe Software](
+http://glencoesoftware.com/2016-08-30-glencoe-software-zeiss-partner-open-source-file-reader-whole-slide.html)
+to add a Java-based JPEG-XR decoder to Bio-Formats. Thanks to extensive,
+fruitful discussions with ZEISS all outputs of this partnership are publicly
+available and all source code licensed identically to other OME projects. Some
+public examples of this ongoing work follow:
```

Comment:
Or 'For some public examples… follow' if you actually mean 'you can follow these' rather than 'examples follow on the page'.